examples/github-workflow/github-workflow.schema.ncl

```diff
@@ -1,2816 +1,2319 @@
 # DO NOT EDIT
 # This file was automatically generated using json-schema-to-nickel
-let js2n
-  = {
-    Always
-      | doc m%"
+let js2n = {
+  Always
+    | doc m%"
         A contract that always succeeds. The same as `Dyn`, but with a more JSON-schema name.
-        
+
```

Comment:
I wonder if our pretty printer the new AST is broken. It seems that it doesn't insert the proper indentation here,  changing the meaning of multi-line strings?

Reply:
Wait, you're entirely right, empty lines aren't part of the indentation computation. So the two are equivalent but the new output is cleaner and probably closer to what you'd write naturally, which sounds obvious retrospectively - I've never needed to write trailing spaces everywhere before inside multiline strings. Sorry for the confusion.